src/fixing/Autofix.ml

```diff
@@ -406,3 +406,45 @@ let apply_fixes_of_core_matches ?dryrun (matches : OutJ.core_match list) =
          let end_ = m.end_.offset in
          Some Textedit.{ path = m.path; start; end_; replacement_text })
   |> apply_fixes ?dryrun
+
+(* Mapping of file path to a list of ranges affected by previous autofixes. *)
+type fixed_lines_env = (Fpath.t, (int * int) list) Hashtbl.t
+
+let make_fixed_lines_env () = Hashtbl.create 13
```

Comment:
For the record the size gets internally rounded to the nearest power of two

Reply:
Didn't know that. In this case I just copied this from the previous implementation, but I usually pick a power of two when I'm doing this from scratch.